src/base/faucet/Faucet.ts

```diff
@@ -0,0 +1,56 @@
+import * as ethers from 'ethers';
```

Comment:
By convention, capitalized file names are for classes, eg. `class Faucet` would be in `Faucet.ts`. In this case we can just call it `faucet.ts` or `lib.ts`.

Reply:
Done